Cargo.toml

```diff
@@ -36,26 +36,26 @@ soroban-token-sdk = { version = "0.8.4", path = "soroban-token-sdk" }
 [workspace.dependencies.soroban-env-common]
 version = "0.0.16"
 git = "https://github.com/stellar/rs-soroban-env"
-rev = "3762edfe52aafc6c0935619211d76d2d05fe87e6"
+rev = "21bd8c2e1636f4646029c9955993874ada369275"
```

Comment:
This points to a https://github.com/stellar/rs-soroban-env/pull/820. Should be updated when that PR is merged.